JSDemos/Demos/DataGrid/FilteringAPI/description.md

```diff
@@ -1 +1,15 @@
-The **DataGrid** widget ships with an&nbsp;integrated filtering API, allowing you apply filters in&nbsp;code. In&nbsp;this example, the grid&rsquo;s filter function is&nbsp;called to&nbsp;filter data based on&nbsp;selections made within the drop-down menu above the **DataGrid**.
\ No newline at end of file
+**DataGrid** has the following API that you can use to filter data:     
+
+- [filter()](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#filter) / [filter(filterExpr)](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Methods/#filterfilterExpr)       
+Gets or applies a filter expression to the grid's [dataSource](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#dataSource).
```

Comment:
It may be read like "gets a filter expression to the grid's... ". I'd use "specifies" or "sets" to avoid this conjunction issue: 

```suggestion
Specifies a filter expression for the grid's [dataSource](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#dataSource).
```